plugins/woocommerce/src/Blocks/AssetsController.php

```diff
@@ -31,7 +33,7 @@ public function __construct( AssetApi $asset_api ) {
 	/**
 	 * Initialize class features.
 	 */
-	protected function init() {
+	public function init() {
```

Comment:
I am aware of a new linting rule `WooCommerce.Functions.InternalInjectionMethod.MissingPublic` which is why this was changed, but I am not 100% sure it applies to this use case. There are other places in codebase where the rule was just ignored. Once this goes public it cannot go back so I'd recommend leaving protected.

Reply:
Good call.  Reverted to protected.